workers/default/xml/xml_generator.py

```diff
@@ -23,7 +23,10 @@ def generate_xml(obj, template_file, target_filepath, params):
     env = Environment(
         loader=FileSystemLoader('resources'),
         trim_blocks=True,
-        lstrip_blocks=True)
+        lstrip_blocks=True,
+        autoescape=select_autoescape(['xml'])
```

Comment:
In der (Jinja2-Doku)[https://jinja.palletsprojects.com/en/2.11.x/api/#autoescaping] (etwas weiter unten), sieht es so aus als würden durch das `select_autoescape(['xml])` nur Dateien escaped, die auf ".xml" enden. Könnte man hier nicht gegebenenfalls auch einfach `True` übergeben, sodass generell escaped wird, egal, was die Endung ist?

Reply:
Ah Mist. Tatsache.